### PR TITLE
Add description fields to all Hyprland key bindings.

### PR DESCRIPTION
**Pull request by ashebanow**
# Pull Request

## Description

This pull request adds description fields to all Hyprland key bindings. This makes the file more readable and easier to understand for new users. But just as importantly, the descriptions will make it possible to use a which-key style help system like the example in [the HyprHook plugin](https://github.com/Hyprhook/Hyprhook/tree/master).

## Type of change

Please put an `x` in the boxes that apply:

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [x] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] **Documentation update** (non-breaking change; modified files are limited to the documentations)
- [ ] **Technical debt** (a code change that does not fix a bug or add a feature but makes something clearer for devs)
- [ ] **Other** (provide details below)

## Checklist

Please put an `x` in the boxes that apply:

- [x] I have read the [CONTRIBUTING](https://github.com/JaKooLit/Hyprland-Dots/blob/main/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] My commit message follows the [commit guidelines](https://github.com/JaKooLit/Hyprland-Dots/blob/main/CONTRIBUTING.md#git-commit-messages).
- [ ] My change requires a change to the documentation.
- [ ] I want to add something in Hyprland-Dots wiki.
- [ ] I have added tests to cover my changes. [NOTE: I didn't find any tests to extend]
- [x] I have tested my code locally and it works as expected.
- [ ] All new and existing tests passed. [NOTE: I didn't find any tests to run]

### CODE REVIEW

File: config/hypr/UserConfigs/UserKeybinds.conf [hunk 1]
```diff
@@ -13,27 +13,27 @@ $UserScripts = $HOME/.config/hypr/UserScripts
...
-bind = $mainMod SHIFT, Return, exec, pypr toggle term # Dropdown terminal
-bind = $mainMod, Z, exec, pypr zoom # Toggle Zoom
+bind = $mainMod SHIFT, Return, "Toggle Floating Terminal", exec, pypr toggle term # Dropdown terminal
+bind = $mainMod, Z, "Toogle Screen Zoom", exec, pypr zoom # Toggle Zoom
 
 # User Added Keybinds
-bind = $mainMod SHIFT, O, exec, $UserScripts/ZshChangeTheme.sh # Change oh-my-zsh theme
-bindn = ALT_L, SHIFT_L, exec, $scriptsDir/SwitchKeyboardLayout.sh # Changing the keyboard layout
+bind = $mainMod SHIFT, O, "Change ZSH Theme", exec, $UserScripts/ZshChangeTheme.sh # Change oh-my-zsh theme
+bindn = ALT_L, SHIFT_L, "Switch Keybolard Layout", exec, $scriptsDir/SwitchKeyboardLayout.sh # Changing the keyboard layout
```
Comment (ashebanow) [inline config/hypr/UserConfigs/UserKeybinds.conf:32]: Typo in "Keybolard" - how did I miss that?!?!

Comment (ashebanow) [reply config/hypr/UserConfigs/UserKeybinds.conf]: Fixed

**Comment by JaKooLit** [comment]
Thanks for this.

however, most of the explanations are already at the end.. See example below

bind = $mainMod, Z, "Toogle Screen Zoom", exec, pypr zoom **# Toggle Zoom**

**Comment by ashebanow** [comment]
For sure, and in fact I used the comment text as a starting point for the
description fields.

But it's a lot more useful to have the descriptions in a form that can be
accessed programmatically.

This PR is just part 1. Second step is a cheatsheet panel in ags that
leverages that data.

**Comment by JaKooLit** [comment]
a great.. understood..

I will review first on my system later. atm just using mobile browser

**Comment by JaKooLit** [comment]
Screenshot_31-Jul_18-25-25_5713

so... I have tried....

The moment I changed it, got so much errors

I am using the hyprland-git version in case you are wondering

**Comment by ashebanow** [comment]
Was out this morning, only just now got time to take a look. Something very
weird is going on. I wasn't seeing the problem, but when I updated my
EndeavourOS via pacman -Syu, it broke. Will dig further, ignore for now.

**Pull request closed without merging**